thoth-export-server/src/xml/doideposit_crossref.rs

```diff
@@ -274,7 +274,7 @@ fn write_chapter_abstract<W: Write>(
         .related_work
         .abstracts
         .iter()
-        .find(|a| a.abstract_type == AbstractType::LONG)
+        .find(|a| a.abstract_type == AbstractType::LONG && a.canonical)
```

Comment:
I think the existing comment above will need updating ("Abstracts must be output in JATS, we simply convert them into JATS by extracting its paragraphs and tagging them with <jats:p>")